keyboards/nazare160/readme.md

```diff
@@ -0,0 +1,22 @@
+# Nazarè Engineering Nazare1-60 QMK Firmware
+
+![nz160](https://i.imgur.com/ZYJgL2G.jpg)
```

Comment:
```suggestion
![nz160](https://i.imgur.com/ZYJgL2Gh.jpg)
```

Reply:
Committed in [19bfe4f](https://github.com/qmk/qmk_firmware/pull/16822/commits/19bfe4fe5844156b33971ff8bc92bc6e5e3c7b20)